SetupDataPkg/Library/ConfigVariableListLib/ConfigVariableListLib.c

```diff
@@ -1,319 +0,0 @@
-/** @file

```

Comment:
This library has other purposes, will remove the commit from this PR and change it separately.

Reply:
This library is left alone, with the exception that the mocked version is no longer needed. Updated in the latest change.